routes.jsx

```diff
@@ -23,6 +23,10 @@ const Latest = () => {
   return <ProjectLoader />;
 };
 
+const Tag = (router) => {
+  return <ProjectLoader tag={router.params.tag} />;
```

Comment:
Does this just work automagically because it hands the API "tag" as a query param?